src/common/containers/landing.js

```diff
@@ -60,40 +59,51 @@ class Landing extends Component {
           </div>
         </div>
 
-        <section className={ `${styles.section} ${styles.sectionNoBorder} ${containerStyles.lightStrip}` }>
-          <div className={ `${styles.row} ${containerStyles.wrapper}` }>
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Scale to millions of installs</li>
-              <li>Available on all clouds and Linux OSes</li>
-              <li>No need for build infrastructure</li>
-            </ListDividedState>
-
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Automatic updates for everyone</li>
-              <li>Roll back versions effortlessly</li>
-              <li>FREE for open source projects</li>
-            </ListDividedState>
+        <section className={ `${styles.section} ${styles.sectionNoBorder}` }>
+          <div className={ `${containerStyles.wrapper}` }>
+            <HeadingTwo className={ styles.heading}>
```

Comment:
Yes, there's commentary in the issue from Jamie about taking this back to the Vanilla WG. I'll rename.